### PR TITLE
add ohlcv subscribe and is_trading api

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: octobot_trading/api/symbol_data.py [hunk 1]
```diff
@@ -87,3 +89,9 @@ def force_set_mark_price(exchange_manager, symbol, price):
 def is_mark_price_initialized(exchange_manager, symbol: str) -> bool:
     return exchange_manager.exchange_symbols_data.get_exchange_symbol_data(symbol).prices_manager.\
         valid_price_received_event.is_set()
+
+
+async def subscribe_to_ohlcv_channel(callback, exchange_id):
```
Comment (Herklos) [inline octobot_trading/api/symbol_data.py:94]: I don't understand why it's in symbol_data api

File: octobot_trading/api/exchange.py [hunk 2]
```diff
@@ -67,6 +67,10 @@ def get_trading_exchanges(exchange_managers) -> list:
             if exchange_manager.is_trading]
 
 
+def is_exchange_trading(exchange_manager) -> bool:
```
Comment (Herklos) [inline octobot_trading/api/exchange.py:70]: :+1:

File: octobot_trading/api/symbol_data.py (hunk 1 above)
Comment (GuillaumeDSM) [reply octobot_trading/api/symbol_data.py:94]: It's data related to a traded symbol, where do you see it otherwise ?

Comment (GuillaumeDSM) [reply octobot_trading/api/symbol_data.py:94]: I'm adding a channels.py api

**Pull request merged**